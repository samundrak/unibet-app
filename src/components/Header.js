import React from 'react';

const Header = () => {
  return (
    <header>
      <div id="logo">Unibet</div>
    </header>
  );
};
export default Header;
