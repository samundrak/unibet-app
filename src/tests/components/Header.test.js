import React from 'react';
import { render } from 'react-dom';
import Header from '../../components/Header';

describe('test Header', () => {
  it('should test snapshot', () => {
    const container = document.createElement('div');
    render(<Header />, container);
    expect(container.innerHTML).toMatchSnapshot();
  });
});
