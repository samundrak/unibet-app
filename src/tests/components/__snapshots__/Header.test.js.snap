// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test Header should test snapshot 1`] = `"<header><div id=\\"logo\\">Unibet</div></header>"`;
